assets/js/admin.js

```diff
@@ -310,6 +310,15 @@ const Admin = {
 
         if (JSON.stringify(defaultElementValue) === JSON.stringify(elementValue)) {
           element.removeAttribute('name');
+        } else if (element.name.endsWith('[]') && '[]' === JSON.stringify(elementValue)) {
```

Comment:
Same check `element.multiple`

Reply:
`defaultValue` have non clear name

Reply:
I think more reable will be 
```JS
if (element.multiple & !elementValue.lengh) {
  elementValue = ''; // Send empty string because empty array not preset at all in HTTP URI
}
```

Reply:
Setting the value doesnt change anything. The form need to be changed instead. I made multiple try in my project.

Reply:
Pls show that you are modified

Reply:
Letter `t` fogotten in property name in example above.

Reply:
> Pls show that you are modified

I invite you to try by yourself instead.

You're just overriding a variable. Not the field value.

Reply:
```JS
if (element.multiple & !elementValue.length) {
  element.value = ''; // Set empty string because empty array not preset at all in HTTP URI
}
```

Reply:
I tried this yesterday but it didnt change anything for the submitted values.

Reply:
Hmm. Could you help what wrote in `configureDatagridFilters`?

Reply:
I'm not sure to understand. If you look for an example, there is one provided in the issue: https://github.com/sonata-project/SonataAdminBundle/issues/7547